features/text-overflow.yml

```
@@ -0,0 +1,10 @@
+name: Text overflow
+description: The `text-overflow` CSS property sets how hidden overflow content is signalled to users. It can be clipped, display an ellipsis (…), or display a custom string.
```

"is signaled to" is a bit indirect, so I'm suggesting "appears to" instead. Also "it" was a unclear to me (the property or content?) and I attempted to resolve the ambiguity.

```suggestion
description: The `text-overflow` CSS property sets how hidden overflow content appears to users. The property can clip content, truncate content with an ellipsis (…), or truncate with a custom string.
```